### PR TITLE
8-GET-api/reviews

### PR DESCRIPTION
**Pull request by RossKeen**

### CODE REVIEW

File: models/reviews.js [hunk 1]
```diff
@@ -27,3 +27,24 @@ exports.updateReview = (review_id, inc_votes) => {
       return rows[0];
     });
 };
+
+exports.selectReviews = (category) => {
+  const queryValues = [];
+  let queryStr = `SELECT reviews.*, COUNT(comment_id) AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id`;
+  if (category) {
+    if (!["social deduction", "dexterity", "euro game", "children's games"].includes(category)) {
+      return Promise.reject({ status: 400, msg: "Invalid category query" });
```
Comment (hevdevs) [inline models/reviews.js]: Nice - have a think about how we could make this dynamic using sql queries to the db to validate the category options available... something to think about going forwards, wouldn't say it's mandatory right here but may be an idea if you're looking to add improvements at a later date!

**Review by hevdevs — APPROVED** [review]
Good to merge Ross! one suggestion for down the road, it may be more relevant for a later ticket too though when we have lots of sort by options to choose from/we know there is functionality for more stuff to be added to the db which could alter these options

**Comment by RossKeen** [comment]
Cool, how does this look as a more dynamic solution?

**Review by hevdevs — APPROVED** [review]
Very nice!! 🎉  good to merge

**Pull request merged**